Gemfile

```diff
@@ -287,6 +287,7 @@ gem 'addressable'
 gem 'bcrypt', '3.1.13'
 gem 'sshkit'
 gem 'validates_email_format_of'
+gem 'validate_url', '~> 1.0.15'
```

Comment:
https://github.com/code-dot-org/code-dot-org/blob/6cad4ce8d6eb32745010662d37ec8980b83beda5/dashboard/app/models/lti_course.rb#L32

Reply:
Very nice catch. How did you spot this?

Reply:
https://drone.cdn-code.org/code-dot-org/code-dot-org/12961/1/6
```
[GEM]/activemodel-6.1.7.7/lib/active_model/validations/validates.rb:121:in `rescue in block in validates': Unknown validator: 'UrlValidator' (ArgumentError)
	from [CDO]/dashboard/app/models/lti_course.rb:32:in `<class:LtiCourse>'
```